src/tests/unit/stream_element_tests.js

```diff
@@ -34,13 +34,13 @@ test("action=append", async () => {
   assert.equal(subject.find("#hello")?.textContent, "Hello Turbo")
 
   subject.append(element)
-  await nextAnimationFrame()
+  await nextRepaint()
```

Comment:
Since these are unit tests, will `nextRepaint()` ever invoke `nextEventLoopTick()`, since the `document.visibilityState` won't ever equal `"hidden"`?